README.md

```diff
@@ -12,26 +19,56 @@ To run a sample:
 
  1. Clone this repo:
 
-    ```bash
-    git clone https://github.com/GoogleCloudPlatform/architecture-center-samples
-    ```
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/architecture-center-samples
+      ```
+
+ 1. Go to the sample directory:
+
+      ```bash
+      cd architecture-center-samples/[SAMPLE]/
+      ```
+
+ 1. [Enable the required APIs](https://console.cloud.google.com/flows/enableapi?apiid=run.googleapis.com,pubsub.googleapis.com,aiplatform.googleapis.com,iam.googleapis.com,storage.googleapis.com,cloudfunctions.googleapis.com,eventarc.googleapis.com,cloudbuild.googleapis.com).
```

Comment:
The list of APIs here is specific to the `gen-ai-rag-vertex-ai-vector-search` sample, right?  If yes, how will we handle this command when we add more samples later ?

Reply:
Whoops! I'll move this to the correct README